python/singa/autograd.py

```diff
@@ -1828,3 +1828,21 @@ def backward(self, dy):
 
 def leakyrelu(x, a=0.01):
     return LeakyRelu(a)(x)[0]
+
+
+class Sub(Operation):
+    def __init__(self):
+        super(Sub, self)._init_()    
```

Comment:
line1835:      super(Sub, self)._init_()  
should be
 super(Sub, self).__init__()

Reply:
fix the code